Good Coding Practices/src/css/styles.css

```diff
@@ -8,21 +8,62 @@ body {
   right: 10px;
   text-align: center;
 }
+
 .btn-arrow {
   width: 30px;
   height: 30px;
 }
 #btn-left {
   margin-right: 32px;
 }
-.circle {
+
+.action {
   width: 100px;
-  height: 100px;
-  border-radius: 50px;
-  background: blue;
+  margin: 5px;
 }
+
 #piece {
   position: absolute;
   top: 100px;
   left: 50%;
+  transition: all .2s;
+}
+
+#piece:hover {
+  background: white;
+}
+
+.circle {
+  width: 100px;
+  height: 100px;
+  border-radius: 50px;
+  background: white;
+  border: white solid 2px;
+}
+
+.blue {

```

Comment:
Good!
Better would be semantic names instead of colors (like 'hot', 'cold', etc.)